### PR TITLE
fix: start.sh linking

### PR DESCRIPTION
**Pull request by vishwa2710**

### CODE REVIEW

File: tools/development/start.sh [hunk 1]
```diff
@@ -58,7 +59,7 @@ if [[ ! -z ${1} ]] && [[ ${1} == "--link" ]]; then
             cp -as /mnt/open-space-toolkit-core/include/OpenSpaceToolkit/Core /usr/local/include/OpenSpaceToolkit/Core; \
             cp -as /mnt/open-space-toolkit-core/src/OpenSpaceToolkit/Core/* /usr/local/include/OpenSpaceToolkit/Core/; \
             ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core.so /usr/local/lib/; \
-            ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core.so.0 /usr/local/lib/;"
+            ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core..so.${project_major_version} /usr/local/lib/;"
```
Comment (kyle-cochran) [inline tools/development/start.sh:62]: extra dot added here?

File: tools/development/start.sh [hunk 2]
```diff
@@ -75,7 +76,7 @@ if [[ ! -z ${1} ]] && [[ ${1} == "--link" ]]; then
             cp -as /mnt/open-space-toolkit-io/include/OpenSpaceToolkit/IO /usr/local/include/OpenSpaceToolkit/IO; \
             cp -as /mnt/open-space-toolkit-io/src/OpenSpaceToolkit/IO/* /usr/local/include/OpenSpaceToolkit/IO/; \
             ln -s /mnt/open-space-toolkit-io/lib/libopen-space-toolkit-io.so /usr/local/lib/; \
-            ln -s /mnt/open-space-toolkit-io/lib/libopen-space-toolkit-io.so.0 /usr/local/lib/;"
+            ln -s /mnt/open-space-toolkit-io/lib/libopen-space-toolkit-io..so.${project_major_version} /usr/local/lib/;"
```
Comment (kyle-cochran) [inline tools/development/start.sh:79]: idem

File: tools/development/start.sh [hunk 3]
```diff
@@ -92,7 +93,7 @@ if [[ ! -z ${1} ]] && [[ ${1} == "--link" ]]; then
             cp -as /mnt/open-space-toolkit-mathematics/include/OpenSpaceToolkit/Mathematics /usr/local/include/OpenSpaceToolkit/Mathematics; \
             cp -as /mnt/open-space-toolkit-mathematics/src/OpenSpaceToolkit/Mathematics/* /usr/local/include/OpenSpaceToolkit/Mathematics/; \
             ln -s /mnt/open-space-toolkit-mathematics/lib/libopen-space-toolkit-mathematics.so /usr/local/lib/; \
-            ln -s /mnt/open-space-toolkit-mathematics/lib/libopen-space-toolkit-mathematics.so.0 /usr/local/lib/;"
+            ln -s /mnt/open-space-toolkit-mathematics/lib/libopen-space-toolkit-mathematics..so.${project_major_version} /usr/local/lib/;"
```
Comment (kyle-cochran) [inline tools/development/start.sh:96]: idem

**Review by kyle-cochran — APPROVED** [review]

File: tools/development/start.sh (hunk 1 above)
Comment (Derollez) [reply tools/development/start.sh:62]: nice catch!

**Pull request merged**